scss/style.scss

```diff
@@ -7,6 +7,7 @@ $o-grid-ie8-rules: 'none';
 @import 'o-grid/main';
 
 @include oFontsInclude(MetricWeb, light); // body
+@include oFontsInclude(MetricWeb, bold); // headings
```

Comment:
I've been meaning to do this for a while. It bumps us up from 150k (~30%) fonts to 200k (~40%). If we're doing this, I might consider nixing Financier Light Italic.